pkg/sqlite/file.go

```diff
@@ -601,7 +618,7 @@ func (qb *FileStore) FindByPath(ctx context.Context, p string) (file.File, error
 		)
 	}
 
-	ret, err := qb.get(ctx, q)
+	ret, err := qb.getMany(ctx, q)
 	if err != nil && !errors.Is(err, sql.ErrNoRows) {
```

Comment:
probably 
```golang
if err!=nil
``` 
is  enough now ?

Reply:
I don't think it would be enough as errors.Is is looking through the whole error chain. In any case I wanted to minimize how much I'm changing the code since surprises can pop up from anywhere as this changing of a ErrNoRows into a nil has shown.